common/src/main/java/com/microsoft/identity/common/internal/cache/ADALOAuth2TokenCache.java

```diff
@@ -56,9 +56,9 @@
 public class ADALOAuth2TokenCache
```

Comment:
Depending on which merges first, there are some conflicts to reconcile between #96 and #103